### PR TITLE
Update env logger

### PR DESCRIPTION
**Pull request by complexspaces**
This PR updates `env-logger` to version `0.9` since it's been out for a few months now (and 0.8 even longer).

The only upstream breaking change is the MSRV being [increased to `1.41`](https://github.com/env-logger-rs/env_logger/releases/tag/v0.8.0). However, while I don't see a MSRV for `pretty_env_logger`, its doubtful it will break any users as Debian stable ships a minimum of 1.41.1 and the majority of users are on newer compiler versions.

Closes https://github.com/seanmonstar/pretty-env-logger/issues/50

### CODE REVIEW

**Comment by complexspaces** [comment]
@djc Thanks for the ping! I've updated this branch to use`0.9` instead.

**Comment by complexspaces** [comment]
@seanmonstar Friendly ping on this PR, if you have the bandwidth available.

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pretty_env_logger"
-version = "0.4.0" # don't forget to update html_root_url
+version = "0.4.1" # don't forget to update html_root_url
```
Comment (seanmonstar) [inline Cargo.toml:3]:
```suggestion
version = "0.4.0" # don't forget to update html_root_url
```

I'd recommend leaving the version bump off:

1. This is likely a breaking change, since `env_logger` is publicly exposed.
2. I prefer version changes to be a standalone commit that can be tagged as "the release".

File: src/lib.rs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #![cfg_attr(test, deny(warnings))]
 #![deny(missing_docs)]
-#![doc(html_root_url = "https://docs.rs/pretty_env_logger/0.4.0")]
+#![doc(html_root_url = "https://docs.rs/pretty_env_logger/0.4.1")]
```
Comment (seanmonstar) [inline src/lib.rs:3]:
```suggestion
#![doc(html_root_url = "https://docs.rs/pretty_env_logger/0.4.0")]
```

**Review by seanmonstar — APPROVED** [review]
Thanks for the PR, and sorry for the delay.

File: Cargo.toml (hunk 1 above)
Comment (complexspaces) [reply Cargo.toml]: > This is likely a breaking change, since env_logger is publicly exposed.

I must have missed where it was exposed publicly, sorry about that. The only crate I saw in the API on docs.rs was `log`.

> I prefer version changes to be a standalone commit that can be tagged as "the release".

Thanks for the info, I'll keep that in mind :thumbsup:

**Comment by praveenperera** [comment]
@complexspaces @seanmonstar now that GitHub actions is setup can we rebase on master and merge this in?

**Comment by complexspaces** [comment]
I've rebased onto the latest `master`, but CI will now need manually run.

**Comment by complexspaces** [comment]
@seanmonstar @praveenperera Sorry for the delay. I've ran `rustfmt` on this branch as CI is now requiring them and that was failing previously. This branch should be ready to merge.

**Comment by jpmckinney** [comment]
Can you create a release that includes this PR?
